client/src/pages/Greetings/Greetings.tsx

```diff
@@ -1,16 +1,66 @@
 import * as React from 'react';
-import { StyledDiv } from '../../assets/styles/styles';
-import Header from '../../components/Header';
+import { Divider, Stack } from '@mui/material';
+
+import { StyledButton } from '../../assets/styles/styles';
+import logo from '../../assets/images/spendr_1.png';
+import picture from '../../assets/images/stonks-nobackground.png';
+import {
+  BodyText,
+  GreetingsDiv,
+  LeftStack,
+  RightBox,
+  WelcomeHeader,
+  ClipArt,
+} from './styles';
 
 /**
  * greetings page component
  * @returns {JSX.Element} - greetings page
  */
 const Greetings = () => {
   return (
-    <StyledDiv>
-      <Header isLoggedIn={false} setIsLoggedIn={() => null} />
-    </StyledDiv>
+    <GreetingsDiv>
+      <LeftStack spacing={'3vh'} alignItems="center">
```

Comment:
Is it necessary to rename all the material components when we add some styling to them? For me it's a bit hard to tell what's what (e.g. BodyText is just styled Typography, LeftStack is a Stack, WelcomeHeader is also a Typography)

Reply:
Its not necessary but its easier to understand when you are reading just the component, also easier to reuse and be consistant, I kinda agreed with victoria if the div styling/option is more than 3 lines then then it should be a style

Reply:
Yeah agreed, the styles should be in a separate file, I just personally find the readability better when it's clear what MUI components are being used without having to see the definition.

If you guys both prefer it then LGTM